package.json

```diff
@@ -29,6 +29,6 @@
   "scripts": {
     "start": "node index.js",
     "watch": "nodemon",
-    "postinstall": "if [ -e .env ]; then source .env; fi && sequelize db:migrate --url $DATABASE_URL"
```

Comment:
Using a `.` as shorthand for `source` will work in every shell I know of, including fish. What in the world does it not work with?

Reply:
source isn't POSIX compliant. The command in master doesn't work on distros running dash as it's POSIX compliant.

Reply:
I learned something new today. Interesting.